packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -5775,4 +5775,39 @@ export default class Meeting extends StatelessWebexPlugin {
 
     return isSuccess;
   }
+
+  /**
+   * disableBNR API
+   * @returns {Promise<Boolean>}
+   * @public
+   * @memberof Meeting
+   */
+  async disableBNR() {
+    LoggerProxy.logger.info('Meeting:index#disableBNR. Disable BNR called');
+    let isSuccess = false;
+
+    try {
+      if (typeof this.mediaProperties === 'undefined' || typeof this.mediaProperties.audioTrack === 'undefined') {
+        throw new Error("Meeting doesn't have an audioTrack attached");
+      }
+      this.mediaProperties.audioTrack = WebRTCMedia.Effects.BNR.disableBNR(this.mediaProperties.audioTrack);
+      const audioStream = MediaUtil.createMediaStream([this.mediaProperties.audioTrack]);
+
+      audioStream.addTrack(this.mediaProperties.audioTrack);
```

Comment:
This is redundant. We already add track in the `MediaUtil.createMediaStream([this.mediaProperties.audioTrack]);` method. Kindly remove

Reply:
Done this change and samples app change as well.